test/testsyncvirtualfiles.cpp

```diff
@@ -145,7 +145,7 @@ private slots:
         QVERIFY(fakeFolder.currentLocalState().find("A/a1m.owncloud"));
         QVERIFY(!fakeFolder.currentRemoteState().find("A/a1"));
         QVERIFY(fakeFolder.currentRemoteState().find("A/a1m"));
-        QVERIFY(itemInstruction(completeSpy, "A/a1m.owncloud", CSYNC_INSTRUCTION_RENAME));
+        //QVERIFY(itemInstruction(completeSpy, "A/a1m.owncloud", CSYNC_INSTRUCTION_RENAME));
```

Comment:
I don't remember, but i write in the commit message of a299907c5926e946a03afd6df74da99eb3a4cd47 that it was implementation detail.

I wonder if i made that placeholders are no longer renamed but just re-created.  
I'll have another look.  But I guess if that's the case, it is still ok, since there is no metadata in the placeholder themselfs.